index.html

```diff
@@ -0,0 +1,148 @@
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-cupcakes.jpg" alt="cupcakes"></img>
+                <div class="card-header">
+                    <div class="card-title">Pasteries and Cupcakes</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-brownies.jpg" alt="brownies"></img>
+                <div class="card-header">
+                    <div class="card-title">Brownies & Dry Cakes</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-jar-cake.jpg" alt="jar-cake"></img>
+                <div class="card-header">
+                    <div class="card-title">Jar Desserts</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-chocolates.jpg" alt="chocolates"></img>
+                <div class="card-header">
+                    <div class="card-title">Chocolates</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-cookies.jpg" alt="cookies"></img>
+                <div class="card-header">
+                    <div class="card-title">Cookies</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/catergory-donuts.jpg" alt="donuts"></img>
+                <div class="card-header">
+                    <div class="card-title">Donuts</div>
+                </div>
+            </div>
+            <div class="card card-basic">
+                <img class="card-content" src="assets/category-gift-hampers.jpg" alt="gift-hampers"></img>
+                <div class="card-header">
+                    <div class="card-title">Gift Hampers</div>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <!-- footer -->
+    <footer class="footer mt-4">
+        <div class="footer-upper">
+            <ul class="footer-quick-links list-group txt-left">
+                <h4 class="headings h4">Quick Links</h4>
+                <li class="list-item"><a href="">Home</a></li>
+                <li class="list-item"><a href="">Cancellation & Refund Policy</a></li>
+                <li class="list-item"><a href="">Terms & Conditions</a></li>
+            </ul>
+            <ul class="footer-contact-links list-group txt-left">
+                <h4 class="headings h4">Get in touch</h4>
+                <li class="list-item"><i class="material-icons txt-bold txt-md pr-2">phone</i><a href="">(+91)
+                        9999999999</a></li>
+                <li class="list-item"><i class="material-icons pr-2">mail_outline</i><a
+                        href="">support@cakery.com</a></li>
```

Comment:
add mail id prefixed with mailto: by which it will open email client application